docs/glyph-inspector.html

```diff
@@ -410,8 +564,19 @@ <h1>Free Software</h1>
     }
     try {
         const data = await file.arrayBuffer();
-        onFontLoaded(opentype.parse(isWoff2 ? Module.decompress(data) : data));
-        showErrorMessage('');
+        let error;
+        try {
+            const base64String = arrayBufferToBase64(data);
+            sessionStorage.setItem('fontData', base64String);
+        } catch (err) {
+            if ( err.message === 'Maximum call stack size exceeded' ) {
```

Comment:
Is this Exception `message` the same on all browser/JSengine ?

Reply:
Good point, probably not. Thinking about it, I think it was caused by an error in the code and didn't occur since, so we might not need this at all.

Reply:
Implemented as a general error message now.